_summaries/qi2016pointnet_1.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: summary
+title: Summary
+paper: qi2016pointnet
+author: slycane9
+score: 9
+---
+* What is the core idea?
+
+Before this paper, state-of-the-art 3D classification/segmentation models all required 3D input to be transformed into voxel grids or multiple image views.  However, this paper introduces PointNets, which only requires point clouds as input to its model.  Using point clouds as input avoids issues with dense image inputs, and avoid irregularities that are difficult to learn from.
+
+* How is it realized (technically)?
+
+![arch](qi2016pointnet_1a.png)
+
+Above is the architecture for PointNet.  The first layers go through an input and feature transformer.  These transformers are themselves mini networks and protect against variance in the point cloud input (ensuring we regard different transformations of the point cloud the same way).
+
+
+Another part of the architecture is the split between a classification network and segmentation network.  This allows sharing of global knowledge with local knowledge.  As you can see in the above figure there is a connection between the two networks.
+
+Finally, the MLPs and Max pooling layers are used to simulate a symmetric function.  A symmetric function is needed to protect against input permutations.  Looking at the symmetric function below, the MLPs represent collection h, and max pooling represents g.
+
```

Comment:
This section on the implementation needs some more work. I would recommend adding information about the actual input, such as its the unordered nature of it the input data and explain a little more about how the symmetric function works and what its doing.